### ISSUE TITLE
Implement update-service operation in firecamp-service-cli

### ISSUE THREAD

**Issue by jazzl0ver** [cloudstax/firecamp#13]
It would be greate to have update-service operation in the cli tool to manage amount of replicas, volumes size, heap size, etc

**Comment by JuniusLuo** [comment]
yes, updating the service and scaling the service will be supported. Increasing the volume size would be a separate feature.

**Comment by cloudstax** [comment]
Updating cassandra heap size and scaling up cassandra service are supported in 0.9.2 release. Updating other configs and scaling other services will be supported in the later release.
